SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1885,7 +1917,36 @@ void USpatialReceiver::OnCommandResponse(const Worker_Op& Op)
 		OnSystemEntityCommandResponse(Op.op.command_response);
 		return;
 	}
+	else if (ComponentId == SpatialConstants::MIGRATION_DIAGNOSTIC_COMPONENT_ID)
+	{
+		check(NetDriver != nullptr);
+		check(NetDriver->Connection != nullptr);
+
+		if (CommandResponseOp.response.schema_type == nullptr)
```

Comment:
Check the status code here instead - and if it's not `WORKER_STATUS_CODE_SUCCESS` I think we can error out.

Reply:
Sure, done.